acinclude.m4

```diff
@@ -15,17 +15,17 @@ AC_ARG_ENABLE(sdltest, [  --disable-sdltest       Do not try to compile and run
   if test x$sdl_exec_prefix != x ; then
      sdl_args="$sdl_args --exec-prefix=$sdl_exec_prefix"
      if test x${SDL_CONFIG+set} != xset ; then
-        SDL_CONFIG=$sdl_exec_prefix/bin/sdl-config
+        SDL_CONFIG=$sdl_exec_prefix/bin/sdl2-config
      fi
   fi
   if test x$sdl_prefix != x ; then
      sdl_args="$sdl_args --prefix=$sdl_prefix"
      if test x${SDL_CONFIG+set} != xset ; then
-        SDL_CONFIG=$sdl_prefix/bin/sdl-config
+        SDL_CONFIG=$sdl_prefix/bin/sdl2-config
      fi
   fi
```

Comment:
Looks like SDL2 is handled by the pkg module now (although the SDL website still suggests `sdl2-config` it looks like that was written 8+ years ago):  
```
PKG_CHECK_MODULES([SDL2], [sdl2 SDL2_net]);
CPPFLAGS="${CPPFLAGS} ${SDL2_CFLAGS}"
LIBS="${LIBS} ${SDL2_LIBS}"
```
I think we could ditch acinclude and move it to to `configure.ac` with other two pkg module calls.  This way probably makes for a more intuitive switch to a newer build system when we do.

Don't have to address in this PR; probably enough change and testing for another PR.

Reply:
I agree, and yeah - let's keept this out of this PR for now; in one of initial versions of SDL2 work I tried to make this part of `acinclude.m4` more readable by using proper bash (e.g. through replacing `if test x$sdl_prefix = x ;` with `if [ -z "$sdl_prefix" ] ;`) but as it turned out, half of the file would need to be rewritten to make it readable (there are like 5 levels of embedding using 3 different languages in one macro in there…).